code/API_definitions/device-swap.yaml

```diff
@@ -17,7 +17,7 @@ info:
 
       # Relevant terms and definitions
 
-      **Device swap**: A Device Swap is a process in which a user's mobile phone number (MSISDN) is associated with a new device (IMEI).
+      **Device swap: A Device Swap is a process in which the association between a user's mobile phone number (MSISDN) and a device (IMEI) is created for the first time or changes for any reasons.
```

Comment:
@jgarciahospital it seems you are not closing the bold sentence

Reply:
fixed! thanks